src/components/historikk/Historikk.tsx

```diff
@@ -68,84 +65,39 @@ const TidligereHendelser = ({
 
 interface HistorikkProps {
   historikkEvents: HistorikkEvent[];
-  tilfeller: TilfellePeriode[];
+  tilfeller: OppfolgingstilfelleDTO[];
 }
 
-type TilfellePerioderMedSkyggeFom = TilfellePeriode & { skyggeFom: Date };
-
-// Dette er en hack for at alle historikkEvents skal få en plassering i et sykefraværstilfellet, selv om de skjer "utenfor".
-const tilfellerSortertMedSkyggeFom = (
-  tilfeller: TilfellePeriode[]
-): TilfellePerioderMedSkyggeFom[] => {
-  const tilfellerSortert = tilfellerNewestToOldest(tilfeller).map(
-    (tilfellePeriode) => ({
-      ...tilfellePeriode,
-      skyggeFom: new Date(),
-    })
-  );
-
-  for (let i = 0; i < tilfellerSortert.length; i += 1) {
-    if (i === tilfellerSortert.length - 1) {
-      tilfellerSortert[i].skyggeFom = new Date(tilfellerSortert[i].fom);
-    } else {
-      tilfellerSortert[i].skyggeFom?.setDate(
-        new Date(tilfellerSortert[i + 1].tom).getDate() + 1
-      );
-    }
-  }
-
-  return tilfellerSortert;
-};
-
 const Historikk = ({
   historikkEvents,
   tilfeller,
 }: HistorikkProps): ReactElement => {
-  const tilfellerSortert = tilfellerSortertMedSkyggeFom(tilfeller);
-  const eventsEtterSisteSykefravaer = historikkEvents.filter(
-    (event: HistorikkEvent) => {
-      return new Date(event.tidspunkt) > new Date(tilfellerSortert[0].tom);
-    }
-  );
-  const eventsForForsteSykefravaer = historikkEvents.filter(
-    (event: HistorikkEvent) => {
-      return (
-        new Date(event.tidspunkt) <
-        new Date(tilfellerSortert[tilfellerSortert.length - 1].fom)
-      );
-    }
-  );
-  const perioderMedEvents = hentSykeforloepMedEvents(
-    tilfellerSortert,
-    historikkEvents
-  );
+  const tilfellerMedEvents = hentTilfelleMedEvents(tilfeller, historikkEvents);
+  const eventUtenforTilfelleList = historikkEvents.filter((event) => {
+    return (
+      tilfeller.filter((tilfelle) => {
+        return isEventInTilfelle(event, tilfelle);
+      }).length === 0
```

Comment:
Er det noen spesiell grunn til at du gjør `filter.length === 0` her, i stedet for `findIndex > 0`, eller `find === undefined`?

Reply:
Kunne også brukt !tilfeller.some((tilfelle) => isEventInTilfelle(event, tilfelle)) 🤓

Reply:
Har byttet til `!some`.